server/routerlicious/packages/lambdas/src/scribe/checkpointManager.ts

```diff
@@ -46,17 +46,28 @@ export class CheckpointManager implements ICheckpointManager {
         const dbOps = pending.map((message) => ({ ...message,
             mongoTimestamp: new Date(message.operation.timestamp) }));
         if (dbOps.length > 0) {
-            await this.opCollection
-                .insertMany(dbOps, false)
-                // eslint-disable-next-line @typescript-eslint/promise-function-async
-                .catch((error) => {
+            let retryCount = 0;
+            let success = false;
+            const wait = async (retryNumber) => new Promise((resolve) => setTimeout(resolve, 10 ** retryNumber));
```

Comment:
I wonder if the retry interval is too low (1ms, 10ms, 100ms). How did we decide on the retry interval? Is there a retry-after header that we could use?

Reply:
In the client, the retry mechanism is defined here: https://github.com/microsoft/FluidFramework/blob/main/packages/loader/driver-utils/src/runWithRetry.ts#L56

Where it starts at 1000ms, doubling each retry. Maybe we should follow suit?

Reply:
I also like that there is a cap in duration in `runWithRetry` at 8s. That way we don't have insanely long retries just hanging around

Reply:
Sounds good, fixed